dist/qs.js

```diff
@@ -323,7 +329,6 @@ var arrayPrefixGenerators = {
 };
 
 var isArray = Array.isArray;
-var split = String.prototype.split;
```

Comment:
I ran `npm run dist` to rebuild the browserified distributable, and it made a bunch of changes unrelated to my PR.

It looks like maybe you forgot to run this before the 6.11.1 release?

Reply:
Nope, it shouldn't *be* run except as part of a release, so the changes to this file should be reverted.

Reply:
Ah okay, thanks, done in 8dc6db802018ef7c05d4bcf63b28b38723677da2

Reply:
Ah okay thanks, done in 8dc6db802018ef7c05d4bcf63b28b38723677da2